cmd/atlas/atlas.go

```diff
@@ -145,7 +145,7 @@ func mongoCLIConfigFilePath() (configPath string, err error) {
 	}
 
 	if configDir, err := config.OldMongoCLIConfigHome(); err == nil { //nolint:staticcheck // Deprecated before fully removing support in the future
-		configPath = fmt.Sprintf("%s/mongocli.toml", configDir)
+		configPath = path.Join(configDir, "mongocli.toml")
 	}
```

Comment:
can't we remove this piece of code?

Reply:
I though about it but we may want to confirm with product I'm going to create a ticket

Reply:
https://jira.mongodb.org/browse/CLOUDP-248054